interfaces/cython/cantera/reaction.pyx

```diff
@@ -277,11 +277,14 @@ cdef class TwoTempPlasmaRate(ArrheniusTypeRate):
 
     .. math::
 
-        k_f = A T_e^b \exp(-\tfrac{E_{a,g}}{RT}) \exp(-\tfrac{E_{a,e}}{RT_e})
-
-    where :math:`A` is the `pre_exponential_factor`, :math:`b` is the
-    `temperature_exponent`, :math:`E_{a,g}` is the `activation_energy`, and
-    :math:`E_{a,e}` is the `activation_electron_energy`.
+        k_f = A T_e^b \exp(-\tfrac{E_{a,g}}{RT}) \exp(\tfrac{E_{a,e}(T_e - T)}{R T T_e})
+
+    where ``A`` is the 
+    `pre_exponential_factor <ArrheniusTypeRate.pre_exponential_factor>`,
+    ``b`` is the `temperature_exponent <ArrheniusTypeRate.temperature_exponent>`,
+    ``E_{a,g}`` is the `activation_energy <ArrheniusTypeRate.activation_energy>`, and
```

Comment:
I don't think this notation quite makes sense. If you want to write the mathematical notation, this should be the LaTeX notation (as it is) but in a ` :math:` block, and if you want to reference the constructor variable name, that should be formatted as code (wrapped in double backticks).

Maybe the best thing to do in cases like `Ea_electron` and `Ea_gas` where the correspondence is imperfect, you could do both, for example:
```
:math:`E_{a,g}` (``Ea_gas``) is the `activation_energy <ArrheniusTypeRate.activation_energy>`